src/io.jl

```diff
@@ -138,6 +139,16 @@ function readg(gfile)
         zlim = [0.0]
     end
 
+    if !eof(f)
+        xdum = take!(token)
+        xdum = take!(token)
+        xdum = take!(token)
+        rhovn = read_array(token,nw)
+        xdum = take!(token)
```

Comment:
I assume rhovn will always be the last quantity in the gfile. Can gfiles exist that have something different than `rhovn`?

Reply:
Might as well merge for now and we'll fix it later if it's an issue